### DISCUSSION TITLE
I can no longer download videos from the FilmBox+ website. "SideDownload error" appears

### DISCUSSION THREAD

**Discussion by VICTORHUGOBRZ** [aclap-dev/video-downloadhelper#94] [Help and Q&A]
### Your question

Hello. I have a FilmBox+ subscription and I was downloading videos normally yesterday, however for certain videos the error message below started to appear

SideDownload error:
[dash @ 000001c7cdef6940] Error when loading first fragment of playlist

[in#0 @ 000001c7cdef6780] Error opening input: Invalid data found when processing input

Error opening input file https://filmboxmedia.tv2zcdn.com/output_videos/videos/22c10d2d-6e3e-4f33-9776-811e51d2153d/dash/manifest.mpd?hdnts=exp=1707946541~acl=/*~hmac=f07ef3ea17e7eb32b48e84b584 77adeb803766f44330a50a075e752727e43d67.

Error opening input files: Invalid data found when processing input
-------------------------------------------------- ------------------------------------------------

I'm using Google Chrome Version 121.0.6167.185 and the Download Helper plugin version 8.2.0.23 with a paid license.

Below are screenshots: https://www.imagebam.com/view/GAAJOB

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: true, browser: chrome
lang: pt-BR
license: accepted
key (only 16 first characters): 128469e9fe6047f7
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\vhpso",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by GrampaWildWilly** [comment]
This is a guess.  Might be relevant, might not.  A number of other users have been encountering false positives on the CoApp in their antivirus & firewall apps.  The solution is to tell both the antivirus & the firewall to make exceptions for the CoApp directory.  The whole directory.  Simpler than doing bits & pieces.  If that's not the answer, then I'll have to leave it to @paulrouget  to help you.

**Reply by paulrouget** [reply]
That would be a tcp error. Doesn't seem to be.

**Comment by paulrouget** [comment]
@VICTORHUGOBRZ can you share the link of a failing video?

**Reply by VICTORHUGOBRZ** [reply]
https://app.filmbox.com/pl/series/naked-attraction-denmark-season-2/naked-attraction-denmark-s02e01

**Reply by paulrouget** [reply]
From which country are you visiting this website?

**Reply by VICTORHUGOBRZ** [reply]
I'm in Brazil, but I use a VPN connection from Poland to access content from that country

**Reply by VICTORHUGOBRZ** [reply]
@paulrouget I'm still waiting for a solution. Thanks

**Comment by Mattmjs** [comment]
@VICTORHUGOBRZ ,You should provide a master manifest & media manifest found in the hit details. How is it done ?
Like this 
![001](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/dbaa8ce8-35ce-443b-b08d-965eef226505)
![details](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/872e44c4-46e0-4742-bdcb-cf48b38f9115)
